hardware_enablement/kmm-release-notes.adoc

```diff
@@ -28,4 +28,251 @@ toc::[]
 * In this release, KMM uses version 1.23 of the Golang programming language to ensure test continuity for partners.
 
 // TELCODOCS-2197
-* You can now schedule KMM pods by defining taints and tolerations. For more information, see xref:../hardware_enablement/kmm-kernel-module-management.adoc#kmm-using-tolerations-for-kernel-module-scheduling_kernel-module-management-operator[Using tolerations for kernel module scheduling].
\ No newline at end of file
+* You can now schedule KMM pods by defining taints and tolerations. For more information, see xref:../hardware_enablement/kmm-kernel-module-management.adoc#kmm-using-tolerations-for-kernel-module-scheduling_kernel-module-management-operator[Using tolerations for kernel module scheduling].
+
+[id="kmm-2-4-RN"]
+== Release notes for Kernel Module Management Operator 2.4
+=== New features
+// TELCODOCS-2343
+* A new feature in this release is the addition of a preflight validation resource in the cluster that you can use to verify kernel modules to be installed on the nodes after cluster upgrades and possible kernel upgrades. Preflight validation also reports on the status and progress of each module in the cluster that it attempts or has attempted to validate. For more information, see xref:../updating/preparing_for_updates/kmm-preflight-validation.adoc#kmm-validation-kickoff_kmm-preflight-validation[Preflight validation for Kernel Module Management (KMM) Modules].
+
+
+// TELCODOCS-2344
+* In this release, a new requirement when creating a kmod image is that both the `.ko` kernel module files and the `cp` binary must be included, which is required for copying files during the image loading process.
```

Comment:
This requirement was added in KMM 2.3. We only extended the documentation to mention the requirement in this release.